calblueprint/ogsc
# Add additional academic fields: internal assessments + standardized testing

* Adds InternalAssessments and StandardizedTesting (name pending input from nonprofit on which types of tests they are tracking) as new profile field keys
* Adds StandardizedTestResult as a ProfileFieldValue type, which tracks a percentile value in addition to a score
* Updates cells and editors for new StandardizedTestResult value type

## How to Test/Use PR feature
(A migration and re-seed is required to run this PR.)

* Open the profiles of seeded players. Verify that internal assessments and standardized testing are appearing in the Academics tab.
* Open the player creation form from an admin user. Verify that you can add, edit, and delete fields for Internal Assessment and Standardized Testing fields.

## PR Checklist

Does your branch (check if true):

## Migrations

* 20210419025239-AddAdditionalAcademicsProfileFields: Adds new profile field keys to profile_field_key enum

## Screenshots

New profile fields in Academics tab:
![image](https://user-images.githubusercontent.com/2977329/115182487-dd16ef00-a08e-11eb-8ba6-a10b92a8b2ba.png)
![image](https://user-images.githubusercontent.com/2977329/115182493-e011df80-a08e-11eb-96a5-c57d5b2b95e5.png)


New creation form for StandardizedTestResult-typed keys:
![image](https://user-images.githubusercontent.com/2977329/115182409-c7a1c500-a08e-11eb-83a8-95b13e41e571.png)


CC: @sydbui
